pre/remap_restart/remap_catchANDcn.py

```diff
@@ -298,8 +298,8 @@ def ask_catch_questions():
 
         {   
             "type": "select",
-            "name": "output:surface::EASE_grid",
-            "message": "Please select EASE grid for the new restart",
+            "name": "output:surface:EASE_grid",
```

Comment:
@weiyuan-jiang, note that I changed the double-colon in `surface::EASE_grid` to `surface:EASE_grid`

Reply:
Good catch. Thanks. This one has not been activated, so it is not debugged